lnurl-withdraw.md

```diff
@@ -36,13 +37,39 @@ Today users are asked to provide a withdrawal Lightning invoice to a service, th
 5. Once accepted by the user, `LN WALLET` sends a GET to `LN SERVICE` in the form of
 
 	```
-	<callback><?|&>k1=<k1>&pr=<lightning invoice, ...>
+	<callback>
+      <?|&> // either '?' or '&' depending on whether there is a query string already in the callback
+       k1=<k1> // the k1 specified in the response above
+      &pr=<lightning invoice> // the payment request generated by the wallet
+      &balanceNotify=<URL> // optional, see note below
 	```
 6. `LN SERVICE` sends a `{"status": "OK"}` or `{"status": "ERROR", "reason": "error details..."}` JSON response and then attempts to pay the invoices asynchronously.
 7. `LN WALLET` awaits for incoming payment if response was successful.
 
 Note that service will withdraw funds to anyone who can provide a valid ephemeral `k1`. In order to harden this a service may require autorization (LNURL-auth, email link etc.) before displaying a withdraw QR.
 
+### `balanceCheck`
+
+This _optional_ property is meant to enable automatic delivery of funds from services to wallets. It works as follows:
+
+  * If `LN WALLET` supports the `balanceCheck` scheme and sees a `balanceCheck` property from a withdraw it performs, it may store that URL locally and call it later (for example, at its next startup, or after 24 hours).
```

Comment:
To complement the above, I'd add: "If the withdrawable amount is zero and the wallet does **not** support `balanceNotify`, then the wallet SHOULD poll the service once per minute for the following 30 minutes."

The idea is that zero means it's expected to go above zero ~soon. But maybe add a special flag for that?